examples/replication/variables.tf

```diff
@@ -13,24 +13,15 @@ variable "prefix" {
 variable "resource_tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
-  default     = []
```

Comment:
`resource_tags` is optional, so put back the default value of `[]`, otherwise it becomes a required variable.